src/sig_stfl/xmss/LICENSE-MIT

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright © 2024 SandboxAQ
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the “Software”), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
Why is there a separate MIT-LICENSE file needed? Which license file now "rules"?

Reply:
I can remove it, do you want me to do so?
MIT license (https://mit-license.org/) has to put copyright holder, so there is it the company name.

Reply:
> I can remove it, do you want me to do so?

I'd prefer just a single LICENSE file documenting all applicable licenses.

Reply:
solved in https://github.com/open-quantum-safe/liboqs/pull/1650/commits/59703957f4ee042f01e833243417f605e902800d